### ISSUE TITLE
Support Swift Mailer 6

### ISSUE THREAD

**Issue by hkdobrev** [OpenBuildings/swiftmailer-css-inliner#27]

**Comment by VahanPWNS** [comment]
Cannot install swiftmailer-css-inliner at all without this! See my result with Symfony 4.0.10 below:
```
    >composer require openbuildings/swiftmailer-css-inliner
    Using version ^0.5.1 for openbuildings/swiftmailer-css-inliner
    ./composer.json has been updated
    Loading composer repositories with package information
    Updating dependencies (including require-dev)
    Your requirements could not be resolved to an installable set of packages.

      Problem 1
        - Installation request for openbuildings/swiftmailer-css-inliner ^0.5.1 -> satisfiable by openbuildings/swiftmailer-css-inliner[0.5.1].
        - Conclusion: remove swiftmailer/swiftmailer v6.0.2
        - Conclusion: don't install swiftmailer/swiftmailer v6.0.2
        - openbuildings/swiftmailer-css-inliner 0.5.1 requires swiftmailer/swiftmailer ^5.1 -> satisfiable by swiftmailer/swiftmailer[v5.1.0, v5.2.0, v5.2.1, v5.2.2, v5.3.0, v5.3.1, v5.4.0, v5.4.1, v5.4.2, v5.4.3, v5.4.4, v5.4.5, v5.4.6, v5.4.7, v5.4.8, v5.4.9].
        - Can only install one of: swiftmailer/swiftmailer[v5.1.0, v6.0.2].
        - Can only install one of: swiftmailer/swiftmailer[v5.2.0, v6.0.2].
        - Can only install one of: swiftmailer/swiftmailer[v5.2.1, v6.0.2].
        - Can only install one of: swiftmailer/swiftmailer[v5.2.2, v6.0.2].
        - Can only install one of: swiftmailer/swiftmailer[v5.3.0, v6.0.2].
        - Can only install one of: swiftmailer/swiftmailer[v5.3.1, v6.0.2].
        - Can only install one of: swiftmailer/swiftmailer[v5.4.0, v6.0.2].
        - Can only install one of: swiftmailer/swiftmailer[v5.4.1, v6.0.2].
        - Can only install one of: swiftmailer/swiftmailer[v5.4.2, v6.0.2].
        - Can only install one of: swiftmailer/swiftmailer[v5.4.3, v6.0.2].
        - Can only install one of: swiftmailer/swiftmailer[v5.4.4, v6.0.2].
        - Can only install one of: swiftmailer/swiftmailer[v5.4.5, v6.0.2].
        - Can only install one of: swiftmailer/swiftmailer[v5.4.6, v6.0.2].
        - Can only install one of: swiftmailer/swiftmailer[v5.4.7, v6.0.2].
        - Can only install one of: swiftmailer/swiftmailer[v5.4.8, v6.0.2].
        - Can only install one of: swiftmailer/swiftmailer[v5.4.9, v6.0.2].
        - Installation request for swiftmailer/swiftmailer (locked at v6.0.2) -> satisfiable by swiftmailer/swiftmailer[v6.0.2].


    Installation failed, reverting ./composer.json to its original content.
```
Symfony SwiftmailerBundle v3.2.2 requires swiftmailer/swiftmailer: ^6.0.1 according to https://packagist.org/packages/symfony/swiftmailer-bundle

:(

**Comment by hkdobrev** [comment]
@VahanPWNS I think the easiest way to support Swift Mailer 6 would be to release 0.6.0 of this package supporting only version 6 without supporting Swift Mailer 5 anymore. Pull request dealing with the Composer version constraint and any breaking changes would be most welcome!

**Comment by VahanPWNS** [comment]
@hkdobrev Took about 3 minutes to install, configure, and successfully use [Emogrifier](https://github.com/MyIntervals/emogrifier) in my project as I've used it before and know how it works. I'll be back next time I need to send emails from a Symfony app though. I'm really close to the launch deadline on my current project so can't spend any more time on this issue :(

**Comment by hkdobrev** [comment]
This is now released in 0.5.2 via #37.
